src/metabase/formatter.clj

```diff
@@ -112,8 +112,10 @@
                              (not grouping) (str/replace #"," ""))]
     (fn [value]
       (if (number? value)
-        (let [scaled-value (* value (or scale 1))
-              decimals-in-value (digits-after-decimal (if percent? (* 100 scaled-value) scaled-value))
+        (let [scaled-value      (cond-> (* value (or scale 1))
+                                  percent?
+                                  (* 100))
```

Comment:
Yea, this is a nice small change that makes the code more sensible :)

Reply:
It also gives consistency with the FE by trimming trailing zeros for percentages. I'm still not sure I like that behavior, but it is consistent.